tests/tensorflow/hooks/test_mirrored_strategy.py

```diff
@@ -198,7 +198,7 @@ def helper_mirrored(
     # Use multiple GPUs by MirroredStragtegy.
     # All avaiable GPUs will be used if `num_gpus` is omitted.
     # if num_devices > 1:
-    distribution = tf.contrib.distribute.MirroredStrategy()
+    distribution = tf.compat.v1.distribute.MirroredStrategy()
```

Comment:
What about tf.MirroredStrategy, 
In code above you check if instance is tf.Mirro... , tf.compat.v1.... 
is there a test case for tf.Mirrored

Reply:
there is tf.compat.v1.mirroredstrategy (tested here) and tf.distribute.mirroredstrategy (tested in https://github.com/awslabs/sagemaker-debugger/blob/d446b26ca11f3ff655f24441af749a26eab560d1/tests/tensorflow/keras/test_keras_mirrored.py#L94). there is no distributed test for TF 2.x eager mode yet. planned for next release